qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/interactor/PlacementQueryAttributesBuilder.kt

```diff
@@ -18,71 +29,87 @@ internal class PlacementQueryAttributesBuilder {
       cachedAttributes: Map<String, JsonObject>
   ) = JsonObject().apply {
     add(VISITOR_ATTRIBUTE_KEY, buildVisitorAttributesJson(deviceId))
-    add(USER_ATTRIBUTE_KEY, customAttributes, cachedAttributes,
-        { convertToUserAttribute(it) }, { buildEmptyUserAttribute() })
-    add(VIEW_ATTRIBUTE_KEY, customAttributes, cachedAttributes,
-        { convertToViewAttribute(it) }, { buildEmptyViewAttribute() })
+    add(USER_ATTRIBUTE_KEY, USER_ATTRIBUTE_SCHEMA, customAttributes, cachedAttributes)
+    add(VIEW_ATTRIBUTE_KEY, VIEW_ATTRIBUTE_SCHEMA, customAttributes, cachedAttributes)
+    addCustomAttributes(customAttributes)
```

Comment:
Name of this method could be more descriptive, so that it's clear that it adds only attributes not existing yet.